typings/index.ts

```diff
@@ -29,7 +29,6 @@ client.on('messageReactionRemoveAll', async message => {
 
 // These are to check that stuff is the right type
```

Comment:
Very, very important
```suggestion
// This is to check that stuff is the right type
```